app/views/content/landing/how-much-do-teachers-get-paid.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "How much do teachers get paid?"
-description: There are great benefits to teaching including a starting salary of at least £30k (or higher in London). Learn more about teachers' salaries and other benefits.
+description: There are great benefits to teaching including a starting salary of at least $salaries_starting_min$ (or higher in London). Learn more about teachers' salaries and other benefits.
```

Comment:
Need to back out this change as variables functionality doesn't work for meta descriptions currently - we will address separately

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/0258a90c-b613-4229-aba2-e46a20ce5e2a)